tests/model_serving/model_server/inference_service_configuration/conftest.py

```diff
@@ -0,0 +1,62 @@
+from typing import Any, Generator
+
+import pytest
+from kubernetes.dynamic import DynamicClient
+from ocp_resources.inference_service import InferenceService
+from ocp_resources.pod import Pod
+
+from tests.model_serving.model_server.inference_service_configuration.constants import (
+    ISVC_ENV_VARS,
+)
+from tests.model_serving.model_server.inference_service_configuration.utils import (
+    update_inference_service,
+)
+from utilities.infra import get_pods_by_isvc_label
+
+
+@pytest.fixture(scope="class")
+def removed_isvc_env_vars(
+    request: pytest.FixtureRequest,
+    admin_client: DynamicClient,
+    ovms_kserve_inference_service: InferenceService,
+) -> Generator[InferenceService, Any, Any]:
+    isvc_predictor_spec_model_env = ovms_kserve_inference_service.instance.spec.predictor.model.get("env", [])
+    isvc_predictor_spec_model_env = [
```

Comment:
Do you want to assert it is not empty list, before the call to update_inference_service()?

Reply:
it will not be an empty list; it may contain env vars that are not part of ISVC_ENV_VARS

Reply:
`.get("env", [])`, then should it be changed to ["env"]?

Reply:
changed to loop only if `isvc_predictor_spec_model_env`, i hope it is clearer